webserver/__init__.py

```diff
@@ -31,6 +33,9 @@ def create_app():
     from kafka_connection import init_kafka_connection
     init_kafka_connection(app.config['KAFKA_CONNECT'])
 
+    # Redis connection
+    app.redis = Redis()
```

Comment:
If it can be done in the way other connections are created, that seems like a better way. There will be a module that we can extend, and it will be consistent with other parts.